bids/layout/layout.py

```diff
@@ -458,9 +460,11 @@ def get(self, return_type='object', target=None, extensions=None,
                 that must be defined in JSON sidecars in order to consider the
                 file a match, but which don't need to match any particular
                 value.
+            absolute_paths (bool): Optional the instance wide option to either
```

Comment:
Docstring is a bit confusing ("the instance wide option"?). Also, it doesn't explain that the default value of `None` uses the layout-level setting.